internal/types/lsp.go

```diff
@@ -1064,6 +1062,7 @@ type ScanIssue struct { // TODO - convert this to a generic type
 	FilePath       string        `json:"filePath"`
 	Range          sglsp.Range   `json:"range"`
 	IsIgnored      bool          `json:"isIgnored"`
+	IsNew          bool          `json:"isNew"`
```

Comment:
I guess, we are adding this in VSCode and IntelliJ for proto version 14, too?

Reply:
yeah we should. I'm thinking if we must do it now tho as soon as we release 14 or is it ok to add it whenever we need it

Reply:
We will need it soon anyway when we implement the tabs